### ISSUE TITLE
[EGOVENTURE] Show Indicator flag of hotspots is ignored in DetailViews

### ISSUE THREAD

**Issue by ThmsKnz** [deep-entertainment/issues#27]
**Describe the bug**
Setting show indicator to false is ignored in scenes shown in DetailView.

**To Reproduce**
Create a scene with a hotspot that has show_indicator = false and display it as DetailView scene.
When hitting 'space' the hotspot indicator is shown although it shouldn't.

**Additional context**
I assume that the reason are missing parentheses in the if clause of the _input function.
By replacing

```
# Hotspot indicator toggle
func _input(event):
	if show_indicator and \
			not DetailView.is_visible or DetailView.is_a_parent_of(self):
```

with

```
# Hotspot indicator toggle
func _input(event):
	if show_indicator and \
			(not DetailView.is_visible or DetailView.is_a_parent_of(self)):
```

the show indicator is also working in DetailView.

**Comment by dploeger** [comment]
I have a fix in branch issue-27. Can you please take a look? Thanks

**Comment by ThmsKnz** [comment]
Thanks for fixing it so quickly. All fine. I've tested this change yesterday and it was OK.
The other hotspot classes don't need adjustment.

**Comment by dploeger** [comment]
Included in 2.1.0
